where.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+CALL "C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -latest -property installationPath
```

Comment:
@jonathanpeppers So I took your advice, this seems to work nicely. Had to put it in a seperate .cmd file though as the
`IN ('')` cmd in the `build-all.cmd` file error when we put this directly in the `IN` command. 
I've tested it on my VM and it works nicely.